great_expectations/checkpoint/checkpoint.py

```diff
@@ -400,3 +408,179 @@ def _get_batches_to_validate(self, batches):
                 batches_to_validate.append(batch)
 
         return batches_to_validate
+
+
+class ActionDicts:
```

Comment:
Nice! I like this - very handy. Only thing that bothers me a little here is that typically, "name" key is arbitrary and user-provided, so it seems weird seeing it in these constants like they're a special GE name. Maybe add "Default" to class name and/or other places?

Reply:
Wow! You know I had **no idea** those were not special! I most certainly will rip those out. Thank you!

Reply:
ok, so I had to backtrack on this because something deep in the data context needs to have a name so I put them back in. Frankly for users of `SimpleCheckpoint` I think it's fine to have them hard coded as if they are special. And, not knowing they were not special, I've gotten by using GE for ages just fine.